azerothcore/azerothcore-wotlk
# fix(dashboard): check if acore user already exists

## Changes Proposed:
-  adding a little check

## Issues Addressed:
![image](https://user-images.githubusercontent.com/519778/102935100-bf9d9400-44a5-11eb-9041-9e21c45ed78b.png)

## How to Test the Changes:
Run the following [dashboard](https://www.azerothcore.org/wiki/Install-with-AC-Dashboard) command:
`./acore.sh db-assembler import-updates`
The second time you run this command without this PR it should return the error above

## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
